Add overview of default local.php

beetofly opened the pull request.

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+Local.php fields overview
+#########
```
This needs to be the full underline length of the title.

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+Local.php fields overview
+#########
+
+Most but not all Mautic configuration can be done from the user interface.
```
You need to use active tense here! Also need to capitalise User Interface.

Maybe:

```suggestion
Most but not all Mautic configuration is managed in the User Interface.
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+#########
+
+Most but not all Mautic configuration can be done from the user interface.
+Either way, the settings live in the local.php file.
```
Let's make it clear that we're talking about a filename.

```suggestion
Either way, the settings live in the ``local.php`` file.
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+
+Most but not all Mautic configuration can be done from the user interface.
+Either way, the settings live in the local.php file.
+Here is the list of the local.php parameters for mautic 4.4.6:
```
```suggestion
Here are the ``local.php`` parameters for Mautic 4.4.6:
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+     - Value
+     - Info
+   * - allowed_extensions
+     - default: An indexed array with the values csv, doc, docx, epub, gif, jpg, jpeg, mpg, mpeg, mp3, odt, odp, ods, pdf, png, ppt, pptx, tif, tiff, txt, xls, xlsx, wav
```
Let's put these into code markup.

```suggestion
     - default: an indexed array with the values ``csv``, ``doc``, ``docx``, ``epub``, ``gif``, ``jpg``, ``jpeg``, ``mpg``, ``mpeg``, ``mp3``, ``odt``, ``odp``, ``ods``, ``pdf``, ``png``, ``ppt``, ``pptx``, ``tif``, ``tiff``, ``txt``, ``xls``, ``xlsx``, ``wav``
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+     - Info
+   * - allowed_extensions
+     - default: An indexed array with the values csv, doc, docx, epub, gif, jpg, jpeg, mpg, mpeg, mp3, odt, odp, ods, pdf, png, ppt, pptx, tif, tiff, txt, xls, xlsx, wav
+     - The "Allowed file extensions" parameter from the Mautic configuration settings (Theme settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
```
We need to use internal linking here.

```suggestion
     - The "Allowed file extensions" parameter from the Mautic configuration settings Theme settings: :ref:`Theme settings`
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+     - The "Allowed file extensions" parameter from the Mautic configuration settings (Theme settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
+   * - anonymize_ip
+     - default: 0 (for 'No')
+     - The "Anonymize IP" parameter from the Mautic configuration settings (Tracking settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
```
```suggestion
     - The "Anonymize IP" parameter from the Mautic configuration tracking settings: :ref:`Tracking settings`
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+     -
+   * - api_enable_basic_auth
+     - default: false
+     - The "Enable HTTP basic auth?" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
```
```suggestion
     - The "Enable HTTP basic auth?" parameter from the Mautic configuration API settings: :ref:`API settings`
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+     - default: false
+     - The "Enable HTTP basic auth?" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
+   * - api_enabled
+     - default: 0 (for 'No')
```
```suggestion
     - default: 0 - for 'No'
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+     - The "Enable HTTP basic auth?" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
+   * - api_enabled
+     - default: 0 (for 'No')
+     - The "API enabled?" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
```
```suggestion
     - The "API enabled?" parameter from the Mautic configuration API settings: :ref:`API settings`
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+     - default: 0 (for 'No')
+     - The "API enabled?" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
+   * - api_oauth2_access_token_lifetime
+     - default: 60 (minutes)
```
```suggestion
     - default: 60 - minutes
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+     - The "API enabled?" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
+   * - api_oauth2_access_token_lifetime
+     - default: 60 (minutes)
+     - The "Access token lifetime (in minutes)" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
```
```suggestion
     - The "Access token lifetime in minutes" parameter from the Mautic configuration API settings: :ref:`API settings`
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+     - default: 60 (minutes)
+     - The "Access token lifetime (in minutes)" parameter from the Mautic configuration settings (API settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
+   * - api_oauth2_refresh_token_lifetime
+     - default: 14 (days)
```
```suggestion
     - default: 14 days
```

RCheesley commented on docs/configuration/local_php_overview.rst:
```diff
@@ -0,0 +1,554 @@
+   * - allowed_extensions
+     - default: An indexed array with the values csv, doc, docx, epub, gif, jpg, jpeg, mpg, mpeg, mp3, odt, odp, ods, pdf, png, ppt, pptx, tif, tiff, txt, xls, xlsx, wav
+     - The "Allowed file extensions" parameter from the Mautic configuration settings (Theme settings): https://mautic-documentation.readthedocs.io/en/latest/configuration/settings.html
+   * - anonymize_ip
```
Let's code-wrap these values so it's clear that it's a setting variable.

```suggestion
   * - ``anonymize_ip``
```

RCheesley reviewed (commented):
I haven't corrected every single instance, but this should give you enough of a head start to implement the correct RST formatting / linking.

Here's some useful resources: https://sublime-and-sphinx-guide.readthedocs.io/ which might be helpful along the way!

RCheesley committed:
Merge pull request #141 from mautic/translations_docs-docs-translations-index-pot--main_pt_BR

Translate 'docs/docs_translations/index.pot' in 'pt_BR'

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-translations-pot--main_pt_BR

RCheesley committed:
Merge pull request #142 from mautic/translations_docs-docs-translations-translations-pot--main_pt_BR

Translate 'docs/docs_translations/translations.pot' in 'pt_BR'

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-contacts-pot--main_pt_BR

RCheesley committed:
Merge pull request #143 from mautic/translations_docs-docs-translations-contacts-pot--main_pt_BR

Translate 'docs/docs_translations/contacts.pot' in 'pt_BR'

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-stages-pot--main_pt_BR

transifex-integration[bot] committed:
Apply translations in pt_BR

100% translated for the source file 'docs/docs_translations/categories.pot'
on the 'pt_BR' language.

transifex-integration[bot] committed:
Apply translations in pt_BR

100% translated for the source file 'docs/docs_translations/plugins.pot'
on the 'pt_BR' language.

RCheesley committed:
Merge pull request #147 from mautic/translations_docs-docs-translations-plugins-pot--main_pt_BR

Translate 'docs/docs_translations/plugins.pot' in 'pt_BR'

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-categories-pot--main_pt_BR

RCheesley committed:
Merge pull request #146 from mautic/translations_docs-docs-translations-categories-pot--main_pt_BR

Translate 'docs/docs_translations/categories.pot' in 'pt_BR'

transifex-integration[bot] committed:
Apply translations in pt_BR

100% translated for the source file 'docs/docs_translations/authentication.pot'
on the 'pt_BR' language.

transifex-integration[bot] committed:
Apply translations in pt_BR

100% translated for the source file 'docs/docs_translations/dashboard.pot'
on the 'pt_BR' language.

beetofly committed:
Update local-php-overview by reason of code review

transifex-integration[bot] committed:
Apply translations in pt_BR

100% translated for the source file 'docs/docs_translations/overview.pot'
on the 'pt_BR' language.

RCheesley committed:
Merge pull request #148 from mautic/translations_docs-docs-translations-authentication-pot--main_pt_BR

Translate 'docs/docs_translations/authentication.pot' in 'pt_BR'

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-overview-pot--main_pt_BR

RCheesley committed:
Merge pull request #150 from mautic/translations_docs-docs-translations-overview-pot--main_pt_BR

Translate 'docs/docs_translations/overview.pot' in 'pt_BR'

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-dashboard-pot--main_pt_BR

RCheesley committed:
Merge pull request #149 from mautic/translations_docs-docs-translations-dashboard-pot--main_pt_BR

Translate 'docs/docs_translations/dashboard.pot' in 'pt_BR'

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-stages-pot--main_pt_BR

RCheesley committed:
Merge pull request #144 from mautic/translations_docs-docs-translations-stages-pot--main_pt_BR

Translate 'docs/docs_translations/stages.pot' in 'pt_BR'

transifex-integration[bot] committed:
Apply translations in pt_BR

100% translated for the source file 'docs/docs_translations/search.pot'
on the 'pt_BR' language.

patrykgruszka committed:
feat: [DPMMA-2095] point leagues documentation

patrykgruszka committed:
feat: [DPMMA-2095] update sidebar

transifex-integration[bot] committed:
Apply translations in pt_BR

100% translated for the source file 'docs/docs_translations/queue.pot'
on the 'pt_BR' language.

RCheesley committed:
Merge pull request #153 from mautic/translations_docs-docs-translations-queue-pot--main_pt_BR

Translate 'docs/docs_translations/queue.pot' in 'pt_BR'

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-search-pot--main_pt_BR

RCheesley committed:
Merge pull request #151 from mautic/translations_docs-docs-translations-search-pot--main_pt_BR

Translate 'docs/docs_translations/search.pot' in 'pt_BR'

patrykgruszka committed:
feat: [DPMMA-2095] cw correct

transifex-integration[bot] committed:
Apply translations in ru

100% translated for the source file 'docs/docs_translations/companies.pot'
on the 'ru' language.

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

RCheesley committed:
Merge pull request #156 from mautic/all-contributors/add-zaharovrd

docs: add zaharovrd as a contributor for translation

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-companies-pot--main_ru

RCheesley committed:
Merge pull request #155 from mautic/translations_docs-docs-translations-companies-pot--main_ru

Translate 'docs/docs_translations/companies.pot' in 'ru'

RCheesley committed:
Merge branch 'main' into feature/overviewLocalphp

favour-chibueze committed:
Merge branch 'main' into TEDU-135-add-stages

favour-chibueze committed:
added images

favour-chibueze committed:
Merge branch 'main' into TEDU-106-contacts-section

favour-chibueze committed:
replace images

favour-chibueze committed:
updated image

favour-chibueze committed:
fixed inconsistent title headings

favour-chibueze committed:
changed duplicated heading

favour-chibueze committed:
fix heading inconsistency

RCheesley committed:
Apply suggestions from code review

tomekkowalczyk committed:
[DPMMA-2108] Add UTM to Asset download #12098

RCheesley committed:
Update assets.rst

tomekkowalczyk committed:
Update docs/components/assets.rst

Co-authored-by: Ruth Cheesley <ruth.cheesley@acquia.com>

tomekkowalczyk committed:
Change section Create Asset

transifex-integration[bot] committed:
Translate authentication.pot in es

100% translated source file: 'authentication.pot'
on the 'es' language.

RCheesley committed:
Merge pull request #160 from mautic/translations_docs-docs-translations-authentication-pot--main_es

Translations for docs/docs_translations/authentication.pot in es

RCheesley committed:
Fix issues with duplicate headings

RCheesley committed:
Small tweak to category explanation and weights.

RCheesley committed:
Merge branch 'main' into TEDU-135-add-stages

RCheesley committed:
Merge pull request #140 from favour-chibueze/TEDU-135-add-stages

TEDU-135 added stages

RCheesley committed:
Merge branch 'main' into fix-asset-headings

RCheesley committed:
Merge pull request #159 from mautic/fix-asset-headings

Fix assets headings

RCheesley committed:
Merge branch 'main' into patch-1

RCheesley committed:
Merge pull request #157 from tomekkowalczyk/patch-1

[DPMMA-2108] Add UTM to Asset download #12098

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

RCheesley committed:
Merge pull request #161 from mautic/all-contributors/add-tomekkowalczyk

docs: add tomekkowalczyk as a contributor for doc

RCheesley committed:
Merge branch 'main' into feature/overviewLocalphp

annamunk committed:
doc: [DPMMA-2044] Editor custom fonts documentation

annamunk committed:
doc: [DPMMA-2044] Vale fixes

transifex-integration[bot] committed:
Translate builders.pot in es

100% translated source file: 'builders.pot'
on 'es'.

transifex-integration[bot] committed:
Translate translations.pot in es

100% translated source file: 'translations.pot'
on 'es'.

transifex-integration[bot] committed:
Translate docs/docs_translations/stages.pot in es

100% translated source file: 'docs/docs_translations/stages.pot'
on 'es'.

RCheesley committed:
Merge pull request #163 from mautic/translations_docs-docs-translations-builders-pot--main_es

Updates for docs/docs_translations/builders.pot in es

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-translations-pot--main_es

RCheesley committed:
Merge pull request #164 from mautic/translations_docs-docs-translations-translations-pot--main_es

Updates for docs/docs_translations/translations.pot in es

RCheesley committed:
Merge branch 'main' into translations_docs-docs-translations-stages-pot--main_es

RCheesley committed:
Merge pull request #165 from mautic/translations_docs-docs-translations-stages-pot--main_es

Updates for docs/docs_translations/stages.pot in es

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

RCheesley committed:
Merge pull request #166 from mautic/all-contributors/add-kukis2107

docs: add kukis2107 as a contributor for translation

RCheesley committed:
Fix location of translated files

RCheesley committed:
Merge pull request #167 from mautic/fix-translations

Fix location of translated files

RCheesley committed:
Commit missing translations

RCheesley committed:
Merge pull request #168 from mautic/push-missing-translations

Add missing translations files

RCheesley committed:
Update README.md

Moongazer committed:
Update command_line_interface.rst

Re-order the table of Mautic commands to make clear what is the command-name and what is an alias.

Next-step: If the community is using commonly more an alias than the actual name, we should refactor the commands by swapping name and alias for a future release (make sure to update these table as well).

Moongazer committed:
Update command_line_interface.rst

Fix some alias typos

RCheesley committed:
Merge pull request #170 from Moongazer/patch-1

Update command_line_interface.rst

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

RCheesley committed:
Merge pull request #171 from mautic/all-contributors/add-putzwasser

docs: add putzwasser as a contributor for review

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

RCheesley committed:
Merge pull request #172 from mautic/all-contributors/add-Moongazer

docs: add Moongazer as a contributor for doc

RCheesley committed:
Add missing translations

RCheesley committed:
Merge pull request #173 from mautic/update-translations

Add missing translations

RCheesley committed:
Merge branch 'main' into DPMMA-2044-custom-fonts

RCheesley committed:
Merge pull request #162 from annamunk/DPMMA-2044-custom-fonts

[DPMMA-2044] Builder custom fonts

escopecz committed:
Starting updating email docs for symfony mailer

escopecz committed:
Documenting SMTP configuration

escopecz committed:
Updating mailer webhook (callback) info for M5

escopecz committed:
Typo fix

escopecz committed:
Updating command to send emails in the background

escopecz committed:
Apply suggestions from code review

Co-authored-by: Ruth Cheesley <ruth.cheesley@acquia.com>

escopecz committed:
Apply suggestions from code review part 2

Co-authored-by: Ruth Cheesley <ruth.cheesley@acquia.com>

escopecz committed:
Adding warning that Symfony transports can be installed only via Composer

escopecz committed:
Adding table with DSN parts explained

escopecz committed:
Replacing the old email send command with the new one

escopecz committed:
Typo fixes

beetofly committed:
Update docs/configuration/local_php_overview.rst

Yes, it is better in lower case.

Co-authored-by: Ruth Cheesley <ruth.cheesley@acquia.com>

patrykgruszka committed:
fix: [DPMMA-2242] rename league to group

patrykgruszka committed:
Merge branch 'main' into DPMMA-2095_point-leagues

RCheesley committed:
Merge pull request #174 from escopecz/symfony-mailer

Symfony mailer docs

RCheesley committed:
Merge branch '5.x' into main

RCheesley committed:
Merge pull request #175 from mautic/main

Merge main into 5.x branch ready for release

RCheesley committed:
Move email settings to config section

RCheesley committed:
Move to top and rename to transport

RCheesley committed:
Add locale definition for translations

RCheesley committed:
Add locale definition for translations

RCheesley committed:
Merge pull request #181 from mautic/fix-translations-locale

Add locale definition for translations

roboparker committed:
add step for MyST Parser

roboparker committed:
add custom fields to chron docs

patrykgruszka committed:
Merge branch '5.x' into DPMMA-2095_point-leagues

patrykgruszka committed:
fix: [DPMMA-2252] group points action and webhook info

patrykgruszka committed:
fix: [DPMMA-2252] fix section title

RCheesley committed:
Update Tx client & re-create translations

RCheesley committed:
Merge branch 'main' into fix-transifex

RCheesley committed:
Remove backup dir, oops!

RCheesley committed:
Merge branch 'fix-transifex' of https://github.com/mautic/user-documentation into fix-transifex

RCheesley committed:
Merge pull request #185 from mautic/fix-transifex

Update Tx client & re-create translations

transifex-integration[bot] committed:
[skip CI] Translate authentication.pot in pt_BR

100% translated source file: 'authentication.pot'
on 'pt_BR'.

RCheesley committed:
Merge pull request #186 from mautic/translations_beba6d19bc494f1f9c9b87ba0939da2b_pt_BR

Updates for project Mautic End User Docs - 2 and lanuage pt_BR on branch main

transifex-integration[bot] committed:
[skip CI] Translate categories.pot in pt_BR

100% translated source file: 'categories.pot'
on 'pt_BR'.

RCheesley committed:
Merge pull request #187 from mautic/translations_beba6d19bc494f1f9c9b87ba0939da2b_pt_BR

Updates for project Mautic End User Docs - 2 and lanuage pt_BR on branch main

RCheesley committed:
Merge branch 'main' into TEDU-106-contacts-section

RCheesley committed:
Merge pull request #152 from patrykgruszka/DPMMA-2095_point-leagues

Point groups

RCheesley committed:
Vale fix

RCheesley committed:
Merge branch 'main' into Update-readme

RCheesley committed:
Merge pull request #169 from mautic/Update-readme

Add detailed instructions on how to make changes

RCheesley committed:
Update README.md

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

RCheesley committed:
Merge pull request #188 from mautic/all-contributors/add-patrykgruszka

docs: add patrykgruszka as a contributor for doc

escopecz committed:
email_transport queue is now called just email

RCheesley committed:
Merge pull request #194 from escopecz/email_transport-to-email

email_transport queue is now called just email

roboparker committed:
Update docs/configuration/cron_jobs.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

roboparker committed:
Update docs/configuration/cron_jobs.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

RCheesley committed:
Fix leftover from rebase :blush:

RCheesley committed:
Remove section that was moved to config

RCheesley committed:
Vale fixes 🙇🏻‍♀️

RCheesley committed:
Missed one 👀🫣

RCheesley committed:
Merge pull request #177 from mautic/refactor-email-dsn-info

Move Email config / DSN settings to Configuration section

RCheesley committed:
Merge branch '5.x' into custom-fields-chron

RCheesley committed:
Merge pull request #183 from diaboloshogunate/custom-fields-chron

add custom fields to cron docs

RCheesley committed:
Update docs/contacts/import_contacts.rst

Co-authored-by: John Linhart <jan@linhart.email>

RCheesley committed:
Merge branch 'main' into TEDU-106-contacts-section

RCheesley committed:
Apply suggestions from code review

RCheesley committed:
Apply suggestions from code review

RCheesley committed:
Apply suggestions from code review - contacts_overview.rst

RCheesley committed:
Apply suggestions from code review - custom_fields.rst

RCheesley committed:
Apply suggestions from code review - frequency_rules.rst

RCheesley committed:
Apply suggestions from code review - import_contacts.rst

RCheesley committed:
Apply suggestions from code review

RCheesley committed:
Apply suggestions from code review - preference_center.rst

RCheesley committed:
Apply suggestions from code review

RCheesley committed:
Apply suggestions from code review - manage_contacts.rst

RCheesley committed:
Update docs/contacts/manage_contacts.rst

RCheesley committed:
Apply suggestions from code review

RCheesley committed:
Update docs/contacts/contacts_overview.rst

RCheesley committed:
Update docs/contacts/contacts_overview.rst

RCheesley committed:
Apply suggestions from code review

RCheesley committed:
Remove import section, link to file

RCheesley committed:
Vale fix

RCheesley committed:
Update docs/contacts/manage_contacts.rst

RCheesley committed:
Fix missing header breaking build

RCheesley committed:
Merge branch 'TEDU-106-contacts-section' of https://github.com/favour-chibueze/user-documentation into TEDU-106-contacts-section

RCheesley committed:
Fix uncommented link fragment

RCheesley committed:
Merge pull request #98 from favour-chibueze/TEDU-106-contacts-section

Tedu-106 - Added contacts section

ItsMly committed:
Update authentication.rst

Added correct configuration for Azure SAML SSO

allcontributors[bot] committed:
docs: update README.md [skip ci]

allcontributors[bot] committed:
docs: update .all-contributorsrc [skip ci]

RCheesley committed:
Merge pull request #198 from mautic/all-contributors/add-Amiyah14

docs: add Amiyah14 as a contributor for doc

ItsMly committed:
Merge branch 'main' into patch-1

ItsMly committed:
Update docs/authentication/authentication.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

ItsMly committed:
Update docs/authentication/authentication.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

ItsMly committed:
Update docs/authentication/authentication.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

ItsMly committed:
Update docs/authentication/authentication.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

ItsMly committed:
Update docs/authentication/authentication.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

ItsMly committed:
Update docs/authentication/authentication.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

ItsMly committed:
Update docs/authentication/authentication.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

ItsMly committed:
Update docs/authentication/authentication.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

ItsMly committed:
Update docs/authentication/authentication.rst

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

RCheesley committed:
Merge pull request #197 from Amiyah14/patch-1

Update authentication.rst

fakela committed:
Add information on the branch to fork

fakela committed:
Update README.md

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

fakela committed:
Update README.md

Co-authored-by: Ruth Cheesley <ruth@ruthcheesley.co.uk>

RCheesley committed:
Merge pull request #201 from mautic/fakela-patch-1

Add information on the branch to fork

RCheesley committed:
fix warning with table syntax

RCheesley committed:
fix exclamation points

RCheesley committed:
Fix tense

RCheesley committed:
Vale fixes

RCheesley committed:
Marketplace link fix

RCheesley committed:
Add reviewdog support

RCheesley committed:
Create .gitpod.Dockerfile

RCheesley committed:
Merge pull request #205 from mautic/#203-reviewdog-support

#203 Add reviewdog support

RCheesley committed:
Update linting.yml

RCheesley committed:
Update linting.yml

https://github.com/errata-ai/vale-action/issues/107

RCheesley committed:
Update .gitpod.Dockerfile

https://github.com/errata-ai/vale-action/issues/107

RCheesley committed:
Merge pull request #206 from mautic/fix-vale-action

Fix vale action

RCheesley committed:
Merge branch 'main' into fix-vale-errors

RCheesley committed:
Merge pull request #204 from mautic/fix-vale-errors

Fix vale errors

RCheesley committed:
Merge branch 'main' into pip-install-myst_parser

RCheesley committed:
Merge pull request #182 from diaboloshogunate/pip-install-myst_parser

add step for MyST Parser

RCheesley committed:
Resolve conflicts

RCheesley committed:
Fix incorrect command

RCheesley committed:
Add missing quotes

RCheesley committed:
Resolve conflicts

RCheesley committed:
Remove language files

RCheesley committed:
Merge pull request #211 from RCheesley/remove-language-files

Remove language files

RCheesley committed:
remove translation files

RCheesley committed:
Merge pull request #213 from RCheesley/remove-translations-5.x

Remove translation files 5.x

escopecz committed:
Merge remote-tracking branch 'upstream/main' into main-2-5

RCheesley committed:
Merge pull request #215 from escopecz/main-2-5

Merge of main into 5.x

RCheesley committed:
Remove deprecated prebuilds for gitpod

RCheesley committed:
File had a space after file extension! :facepalm:

RCheesley committed:
Merge pull request #219 from mautic/fix-build-fail

Fix build fail

RCheesley committed:
Merge pull request #220 from mautic/main

Merge Main to 4.x

RCheesley committed:
Merge branch 'main' into feature/overviewLocalphp

RCheesley committed:
Merge branch '4.x' into feature/overviewLocalphp

adiati98 committed:
fix: remove doctrine schema update command

RCheesley committed:
Merge pull request #403 from adiati98/fix/issue_399_4.x

Remove update command from 4.x docs

adiati98 committed:
Merge branch '4.x' into feature/overviewLocalphp

adiati98 committed:
update linting.yml

adiati98 committed:
update .gitpod.Dockerfile

adiati98 committed:
update requirements.txt

adiati98 committed:
update gitpod.yml

adiati98 committed:
update .readthedocs.yaml

adiati98 committed:
add .ddev/commands/web/build-docs

adiati98 committed:
add .ddev/web-build/Dockerfile.sphinx

adiati98 committed:
add .ddev/web-build/requirements.txt

adiati98 committed:
add .ddev/config.yml

adiati98 committed:
add .devcontainer/devcontainer.json

adiati98 committed:
add .devcontainer/setup-project.sh

adiati98 committed:
move vocab/Mautic to config folder and rename Vocab to vocabularies

adiati98 committed:
update build-docs file

adiati98 committed:
rename config.yml to config.yaml

adiati98 committed:
update .gitignore

adiati98 committed:
update conf.py

adiati98 committed:
fix: link in maxmind_ccpa.py

RCheesley committed:
Merge pull request #440 from adiati98/fix/build-prose-failed

Fix: Read the Docs, build and prose failed in branch `4.x`, and add DDEV configuration

adiati98 committed:
Merge branch '4.x' into feature/overviewLocalphp

adiati98 committed:
fix: overflow text in the last table's column

adiati98 committed:
fix: broken links
